state/wwwpublic/support-migrid.org.html

```diff
@@ -204,18 +204,23 @@ <h4 class="staticpage">How much storage space do I have on MiGrid?</h4>
                             <p>
                                 There's no space quota as such on the general MiGrid
                                 storage, but we do recommend using our ERDA/IDMC services
```

Comment:
Just noticed this leftover reference to IDMC, which is retired. I'll search and remove during merge :)